index.html

```diff
@@ -35,6 +38,14 @@ <h1 align="center">Safe Research</h1>
             <a href="https://github.com/ennisjo"><img src="https://avatars.githubusercontent.com/u/25203220?s=200" width="100" height="100" alt="ennisjo" /></a>
             <a href="https://github.com/remedcu"><img src="https://avatars.githubusercontent.com/u/30735581?s=200" width="100" height="100" alt="remedcu" /></a>
         </div>
+        <br /><br />
+        <div align="center">
+          <a href="https://github.com/safe-research" target="_blank">Find us on GitHub</a>.
+          <script>
+            document.write('<a href="mailto:' + 'research' + '@' + 'safe.dev">Send us a message</a>');
+          </script>.
```

Comment:
Why do you need to `document.write` it, and can't just:

```suggestion
          <a href="mailto:research@safe.dev">Send us a message</a>.
```

Is it just to obfuscate for scrapers?

Reply:
> Is it just to obfuscate for scrapers?

Exactly. I don't know how effective it is since I'd assume that a smart scraper would not be fooled by this. But since it might help with some scrapers, I'd leave it in. (Or actually switch to the workaround that Shebin suggested above.)